sql/migrations/20200417194325_world.sql

```diff
@@ -0,0 +1,110 @@
+DROP PROCEDURE IF EXISTS add_migration;
+delimiter ??
+CREATE PROCEDURE `add_migration`()
+BEGIN
+DECLARE v INT DEFAULT 1;
+SET v = (SELECT COUNT(*) FROM `migrations` WHERE `id`='20200417194325');
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20200417194325');
+-- Add your query below.
+
+
+-- update Desperate Prayer (5635) to Priestess Josetta (377)
+UPDATE `creature_questrelation` SET `id` = 377 WHERE `quest` = 5635;
+
+-- update Desperate Prayer (5637) to Maxan Anvol (1226)
+UPDATE `creature_questrelation` SET `id` = 1226 WHERE `quest` = 5637;
+
+-- update Desperate Prayer (5640) to Priestess Alathea (11401)
+UPDATE `creature_questrelation` SET `id` = 11401 WHERE `quest` = 5640;
+
+-- add missing quest Desperate Prayer (5640)
+INSERT INTO `quest_template` (`entry`, `ZoneOrSort`, `MinLevel`, `QuestLevel`, `RequiredClasses`, `RequiredRaces`, `QuestFlags`, `ExclusiveGroup`, `Title`, `Details`, `Objectives`, `OfferRewardText`, `RequestItemsText`, `RewXP`, `RewMoneyMaxLevel`, `RewSpell`, `CompleteEmote`) VALUES 
+(5640, -262, 10, 10, 16, 5, 8, 5634, 'Desperate Prayer', 'Praise be to Elune, and to your Light, $n.$b$bAbout this time, many of our order learn abilities reflecting their faith in Elune. If knowledge serves me, I believe the same holds true for your order also.$b$bWhen your business in Darnassus is complete, I would seek out one of your High Priestesses for further training., and may the Light be with you.', 'Speak to High Priestess Laurena in Stormwind.', "I'm glad you've arrived, $n. There are many things we will have to discuss, but more importantly is your training in the ways of the Light.$b$bThere are lessons all servants of the Light must learn. If you are ready, we'll begin discussing some of those now.", NULL, 210, 120, 19338, 1);
+
+-- add quest Desperate Prayer (5640) to Priestess Alathea (11401)
+INSERT INTO `creature_questrelation` (`id`, `quest`, `patch_min`, `patch_max`) VALUES

```

Comment:
You should add creature_involvedrelation too because you can't turn in the quest otherwise. Quest relation is the quest giver, Involved relation is the turn in npc.